### PR TITLE
Get all files

### PR DESCRIPTION
**Pull request by harrison4anewbe**
Modify the get source data function, get all files in one time.

### CODE REVIEW

**Comment by harrison4anewbe** [comment]
Can not get some source code in projects libyang, llhttp, miniz, because the paths in JSON do not exist in docker images.
[https://datacorrupted.atlassian.net/wiki/spaces/FUN/pages/40861719/CODING+Function+matching+and+source+code+retrieve](url)

File: scripts/demangle.py [hunk 1]
```diff
@@ -10,62 +12,56 @@
 from source_code import *
 
 
-# get source code from docker
-def get_source_from_docker(
-    file_path: str, file_name: str, proj_name: str
-) -> Optional[str]:
-    # Create a container from the image
+# Get all files from docker
+def copy_files_from_docker(proj_name: str, output_path: str) -> bool:
```
Comment (EYH0602) [inline scripts/demangle.py:16]: please add a doc-string saying the return value is whether copying success

File: scripts/demangle.py [hunk 2]
```diff
@@ -10,62 +12,56 @@
...
-                    code_content = None
-                    error(f"File  {file_path}  NOT found\n")
-    # Stop and remove the container
     container.stop()
     container.remove()
-    return code_content
+    return True
+
+
+# get source code from docker
```
Comment (EYH0602) [inline scripts/demangle.py:40]: this function is getting path from `OSSFUZZ_SCRIPTS/output/proj_name/codes`, not docker right?

**Review by EYH0602 — APPROVED** [review]

File: scripts/demangle.py (hunk 2 above)
Comment (harrison4anewbe) [reply scripts/demangle.py:40]: You mean this function `get_source_code_path`? If so, yes, it gets the path from local.

**Pull request merged**